### PR TITLE
no more array insensitive dots as recent 8th update corrects array to…

### PR DESCRIPTION
**Pull request by axtens**
… string conversion issue

So I thought this had been dealt with. But I was just trying to update my repo to the upstream and go sync error

### CODE REVIEW

File: exercises/practice/grains/libs/exercism/test [hunk 1]
```diff
@@ -12,20 +12,6 @@ true var, run-test
...
-  then 
-  .
-;
-
-: test-passed \ s x x --   \\ test name, expected value, actual value
-    2drop
-    1 tests-passed n:+!
-    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
-;
-
```
Comment (glennj) [inline exercises/practice/grains/libs/exercism/test:28]: Too much deleted for grains.

File: exercises/practice/pythagorean-triplet/libs/exercism/test [hunk 2]
```diff
@@ -12,20 +12,6 @@ true var, run-test
...
-  then 
-  .
-;
-
-: test-passed \ s x x --   \\ test name, expected value, actual value
-    2drop
-    1 tests-passed n:+!
-    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
-;
-
```
Comment (glennj) [inline exercises/practice/pythagorean-triplet/libs/exercism/test:28]: too much deleted for pythagorean-triplet

File: exercises/practice/series/libs/exercism/test [hunk 3]
```diff
@@ -12,20 +12,6 @@ true var, run-test
...
-  then 
-  .
-;
-
-: test-passed \ s x x --   \\ test name, expected value, actual value
-    2drop
-    1 tests-passed n:+!
-    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
-;
-
```
Comment (glennj) [inline exercises/practice/series/libs/exercism/test:28]: too much deleted for series

File: exercises/practice/sieve/libs/exercism/test [hunk 4]
```diff
@@ -12,20 +12,6 @@ true var, run-test
...
-  then 
-  .
-;
-
-: test-passed \ s x x --   \\ test name, expected value, actual value
-    2drop
-    1 tests-passed n:+!
-    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
-;
-
```
Comment (glennj) [inline exercises/practice/sieve/libs/exercism/test:28]: sieve too

File: exercises/practice/sum-of-multiples/libs/exercism/test [hunk 5]
```diff
@@ -12,20 +12,6 @@ true var, run-test
...
-  then 
-  .
-;
-
-: test-passed \ s x x --   \\ test name, expected value, actual value
-    2drop
-    1 tests-passed n:+!
-    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
-;
-
```
Comment (glennj) [inline exercises/practice/sum-of-multiples/libs/exercism/test:28]: sum-of-multiples too

**Review by glennj — CHANGES_REQUESTED** [review]

**Comment by axtens** [comment]
Okay, so what's the best way of handling this?

**Comment by glennj** [comment]
https://github.com/exercism/8th/blob/main/libs/README.md

**Comment by glennj** [comment]
Did I misunderstand "what's the best way of handling this"? Make the change in `libs/exercism/test` and copy it to the exercises. That way, you minimize editing errors.

**Comment by axtens** [comment]
I'm not disputing your advice. It's correct. The difficulty is with my understand of the situation. I have a test library in `8th/libs/exercism/test` that has, on line 67-68
```
   \ otherwise fall back to 'lazy evaluation'
   l: =
```
and it is this line that the other exercises are bombing on. So will copying this file to those exercises fix the problem?

**Comment by axtens** [comment]
I didn't write the test library.

**Comment by glennj** [comment]
That came from Ron:
https://8th-dev.com/forum/index.php?topic=2745.msg15984#msg15984

I see in the words list:

l: | --
-- | --
  | **IMMEDIATE** _needs utils/latebind_  Makes the next word "delayed lookup". Compiles the string <name> into the word currently being compiled. At runtime, every time the code is invoked, it looks up the word with that name. So l: + will add two numbers, or concatenate two strings, using the same code.

Try adding `needs utils/latebind` to the test file

**Pull request closed without merging**